src/Tests/dotnet-new.Tests/DotnetNewInstantiateTests.Approval.cs

```diff
@@ -362,11 +362,10 @@ public Task CanInstantiateTemplate_ConditionalProcessing()
             string[] actualFiles = Directory.GetFiles(workingDirectory);
 
             return Task.WhenAll(
-                actualFiles.Select(
+                actualFiles.Where(f => Path.GetExtension(f).Equals(".txt")).Select(
```

Comment:
this test should be rewritten as verify test

Reply:
Fully agree.
Not in the scope of this PR though. I added subitem to the tracking feature https://github.com/dotnet/templating/issues/3868